### ISSUE TITLE
AuthorizationError: Missing or invalid 'refresh_token' field in v2.2.0

### ISSUE THREAD

**Issue by leandrakelly** [sergiodxa/remix-auth-oauth2#111]
When upgrading from version 2.1.0 to 2.2.0 of remix-auth-oauth2, I encountered an error during the OAuth callback process. The error message states: "AuthorizationError: Missing or invalid 'refresh_token' field". Importantly, no changes were made to our Auth0 configuration or our application code; the only change was updating the library version.

**Comment by leandrakelly** [comment]
After further investigation, I've identified the root cause and have some insights to share.

1. Missing 'offline_access' Scope:
The problem stems from the 'offline_access' scope not being automatically included in the authorization request. While I haven't confirmed this yet, it's possible that in previous versions of oslojs, this scope might have been added by default.

As a test, I tried explicitly adding the 'offline_access' scope, which resolved the issue:

```typescript
const scopes = new Set(['offline_access', ...(this.options.scopes || [])]);
authorizationURL.addScopes(...Array.from(scopes));
```

This ensures that the 'offline_access' scope is always included, along with any other scopes specified in the options. However, I want to emphasize that this was just a test, not a proposed resolution.

2. Temporary Fix - Manual Scope Addition in Auth0Strategy:
As a temporary decision to fix the problem for those who need it immediately, I've found that manually adding the 'offline_access' scope when initializing the Auth0Strategy works:

```typescript
new Auth0Strategy(
  {
    // ... other options
    scopes: ['openid', 'email', 'profile', 'offline_access'],
  },
  // ... callback function
)
```

It's crucial to note that this issue is causing significant problems for Auth0 applications. Without the refresh token, these applications are now unable to complete the login callback process successfully. This raises an important question: Is this behavior intentional? In previous versions, we didn't need to explicitly set this scope.
